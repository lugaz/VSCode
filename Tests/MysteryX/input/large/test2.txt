getRear
insertRear 161
getFront
getRear
deleteFront
insertFront 104
deleteFront
deleteFront
getFront
insertFront 42
insertRear 240
deleteFront
getRear
deleteRear
deleteRear
getRear
insertRear 860
deleteFront
getRear
deleteRear
getRear
deleteRear
insertRear 650
deleteFront
getRear
getFront
insertFront 677
deleteRear
getRear
deleteRear
insertRear 981
deleteFront
deleteRear
deleteRear
insertFront 981
getFront
insertRear 22
insertFront 952
getRear
deleteFront
insertFront 315
deleteRear
insertRear 340
getFront
deleteFront
deleteFront
getRear
deleteFront
insertFront 525
insertFront 771
deleteFront
getFront
getFront
deleteFront
insertFront 373
deleteFront
deleteFront
insertFront 657
insertRear 763
deleteRear
getFront
getFront
insertRear 944
insertRear 107
deleteRear
getFront
getRear
insertRear 553
getRear
getFront
deleteRear
deleteFront
deleteFront
getRear
deleteRear
deleteFront
insertRear 357
deleteFront
getFront
deleteFront
deleteFront
getFront
deleteRear
insertRear 918
insertFront 921
deleteRear
insertFront 165
getRear
getRear
deleteRear
insertRear 865
deleteFront
deleteRear
deleteRear
insertRear 432
getFront
getFront
getRear
deleteFront
getRear
insertFront 129
insertFront 981
getFront
insertFront 431
getRear
deleteRear
deleteFront
getFront
deleteRear
deleteFront
insertFront 427
getFront
insertRear 445
getRear
deleteRear
getRear
getFront
getFront
deleteRear
getFront
getRear
deleteFront
getRear
insertRear 186
deleteFront
deleteRear
deleteFront
insertRear 875
getFront
getFront
insertRear 665
getFront
insertFront 536
getFront
getRear
getRear
getFront
insertFront 653
insertFront 246
deleteRear
getFront
insertFront 173
insertFront 492
deleteRear
insertFront 249
getFront
insertRear 460
getFront
insertRear 14
insertRear 790
deleteFront
insertRear 289
insertFront 689
deleteFront
getRear
insertFront 375
getRear
deleteFront
getRear
insertRear 709
insertFront 122
getFront
insertRear 82
getFront
getRear
insertRear 886
insertFront 721
getFront
getFront
insertRear 844
getRear
getRear
insertRear 671
getFront
insertFront 17
insertFront 709
getFront
insertFront 245
insertRear 417
deleteRear
deleteFront
getRear
deleteRear
insertRear 495
getFront
getFront
insertRear 773
deleteFront
insertFront 233
insertRear 626
deleteRear
insertRear 656
insertRear 881
insertRear 407
getRear
getRear
getFront
insertRear 645
insertFront 720
insertFront 616
deleteRear
deleteFront
deleteRear
getFront
insertFront 987
insertRear 333
deleteFront
deleteRear
insertFront 141
insertFront 959
getRear
getFront
insertFront 147
insertRear 175
insertRear 954
getRear
deleteRear
getFront
getFront
insertFront 736
insertRear 7
getRear
deleteFront
deleteFront
insertRear 350
deleteRear
insertFront 822
insertFront 713
insertRear 433
deleteRear
getRear
deleteRear
deleteRear
insertFront 558
insertFront 850
deleteFront
getRear
getFront
deleteFront
deleteRear
deleteFront
deleteRear
insertRear 346
deleteFront
insertFront 648
deleteRear
getRear
insertRear 916
deleteRear
deleteFront
deleteFront
deleteFront
insertFront 896
deleteFront
deleteRear
deleteFront
deleteRear
insertRear 173
deleteRear
getRear
deleteRear
deleteRear
insertFront 588
deleteFront
insertFront 515
insertFront 310
insertRear 125
deleteFront
insertFront 887
deleteRear
getRear
insertFront 152
getRear
insertFront 182
insertRear 791
insertFront 845
insertFront 280
deleteRear
deleteFront
deleteRear
deleteRear
getRear
deleteRear
insertRear 911
insertRear 842
insertFront 865
getFront
getFront
deleteRear
insertFront 618
getFront
insertFront 206
deleteRear
insertFront 661
insertFront 105
deleteRear
getFront
deleteFront
getRear
getFront
insertRear 322
deleteRear
getRear
getRear
getFront
insertRear 239
insertFront 258
getRear
getRear
deleteRear
getRear
deleteRear
deleteFront
deleteRear
getFront
deleteRear
insertRear 361
deleteRear
insertRear 536
insertRear 223
deleteRear
deleteRear
getFront
insertRear 282
getFront
getFront
insertFront 814
deleteFront
insertFront 772
