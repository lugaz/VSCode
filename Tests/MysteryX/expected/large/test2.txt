-1
161
161
161
104
-1
-1
42
240
240
-1
-1
860
-1
-1
-1
-1
650
-1
-1
677
-1
-1
981
-1
-1
981
22
952
22
315
315
981
340
340
771
525
525
525
373
-1
763
657
657
107
657
944
553
657
553
657
944
-1
-1
-1
357
-1
-1
-1
-1
-1
918
921
921
921
165
865
-1
432
432
432
432
-1
981
129
129
431
981
981
-1
427
445
445
427
427
427
427
-1
-1
-1
-1
186
-1
-1
875
875
875
536
665
665
536
665
246
875
249
249
249
689
289
289
375
289
122
122
82
721
721
844
844
721
709
417
245
671
671
709
709
709
626
407
407
233
645
616
407
720
987
333
881
959
954
954
147
147
7
736
147
350
433
7
7
175
850
881
558
558
881
713
656
822
346
773
916
648
959
141
896
773
720
495
173
844
844
886
588
310
125
82
82
791
280
82
709
289
289
865
865
842
618
911
790
105
105
14
661
322
14
14
661
239
239
239
14
14
258
460
661
536
361
223
536
661
661
661
814
